src/components/EditNotesCard.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/prop-types */
+import {useRef, useState} from "react";
+import "../styles/component/editnotes.css"
+import "../styles/utils/variable.css"
+import { useNotes } from "../context/notes-context"
+import { v4 as uuid } from "uuid"
+
+export const EditNotesCard = () =>{
+  const editNotesForm = useRef(null);
+  const {notesState,dispatchNotes} = useNotes();
+  console.log(notesState.setEditBox);
+  console.log(notesState);
+
+  const [errorText,setErrorText] = useState("");
+
+  const handleSaveForm = ()=>{
+    const form = editNotesForm.current;
+      if(form["editTitle"].value.length>0 && form["editBody"].value.length>0){
+          setErrorText("");
+          return {_id: uuid(),title:form["editTitle"].value,body:form["editBody"].value,editBoxStatus:"hide-edit-box",isPinned:notesState.isPinned};
+    }
```

Comment:
Do implement the use of code formatter, will make reviewing and code readability much easier

Reply:
Thanks for your Review. will implement changes in new commit.